### PR TITLE
Updates operator to go to target poses, while driver still finishes them

### PR DESCRIPTION
**Pull request by jnallard**
Creates the commands such that the operator moves the arms into the prep position (prep position is target position for cubes), but the driver will still execute the score command.

Also made it so the tester controller doesn't constant throw errors unless we enable it.

### CODE REVIEW

**Comment by jnallard** [comment]
(I don't know why this got closed?)

File: src/main/java/frc/robot/commands/Score.java [hunk 1]
```diff
@@ -9,29 +9,33 @@
...
+  public Score(Arm arm, Supplier<ArmPose> prepPoseSupplier, Supplier<ArmPose> targetPoseSupplier, int endWaitSeconds) {
     if (prepPoseSupplier != null){
       addCommands(new MoveArm(arm, prepPoseSupplier));
     }
     Command goToScorePosition = new MoveArm(arm, targetPoseSupplier);
-    Command release = new UnGrip(arm).withTimeout(1);
+    Command release = new InstantCommand(() -> arm.setGripperMode(GripperMode.slowUngrip));
     addCommands(goToScorePosition, release);
     if (prepPoseSupplier != null){
       addCommands(new MoveArm(arm, prepPoseSupplier));
```
Comment (Kenneth-Olibrice) [inline src/main/java/frc/robot/commands/Score.java:32]: Why do we do this twice?

Comment (jnallard) [reply src/main/java/frc/robot/commands/Score.java:32]: We want to go to the scoring prep pose, go to the scoring pose, turn on the ungrip, then go back to the prep pose.
This is specifically for cones. For cubes, the prep pose and scoring pose are the same, so it should complete immediately

Comment (jnallard) [reply src/main/java/frc/robot/commands/Score.java:32]: (And for the first prep pose, hopefully the operator has already gotten us there, so it should be a no-op too)
(no-op is a programming term for no-operations, or no work needed)

**Comment by jnallard** [comment]
We can probably close this since we will have a new scoring method with GripperV2. 
(But I'll leave it open until we confirm we don't)

**Pull request closed without merging**